pkg/metrics/data/snapshotter.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package data
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	snapshotIDLabel = "snapshot_id"
+	beginLabel      = "begin"
+	endLabel        = "end"
+)
+
+var (
+	PrepareTime = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "snapshotter_prepare_snapshot_time_milliseconds",
+			Help: "The time to prepare a snapshot.",
+		},
+		[]string{snapshotIDLabel, beginLabel, endLabel},
+	)
+
+	MountTime = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "snapshotter_mounts_snapshot_time_milliseconds",
+			Help: "The time to mount a snapshot.",
+		},
+		[]string{snapshotIDLabel, beginLabel, endLabel},
+	)
+
+	RemoveTime = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "snapshotter_remove_snapshot_time_milliseconds",
+			Help: "The time to remove a snapshot.",
+		},
+		[]string{snapshotIDLabel, beginLabel, endLabel},
+	)
```

Comment:
Can we also add a metric for Cleanup interface?

Reply:
Done.